lib/src/core/parameter/parameter_object.dart

```diff
@@ -0,0 +1,7 @@
+class FetchMovieParams {
+  final int? page;
+  final int? genre;
+  final String? endpoint;
+
+  FetchMovieParams({this.page, this.genre, this.endpoint});
```

Comment:
trailing comma

Reply:
The Fetch at start of the name is not necessary, what happened if in the future you want to use this same params not only to made a GET, the name should not refer to an action.
You can use MovieParams or ApiParams

Reply:
good point, fixed!